content/kots-cli/download.md

```diff
@@ -5,4 +5,24 @@ title: kots download
 weight: 90120
 ---
 
-Once an application is running and has the Admin Console (kotsadm) deployed with it, you can run `kots download` to retrieve a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation.
+Retrieves a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation. Requires a running KOTS application with the Admin Console (kotsadm).
+
+### Usage
+```bash
+kubectl kots download [app-slug] [flags]
```

Comment:
does this require [namespace] as well

Reply:
namespace is included in the [flags]. Strictly speaking, not a required argument, so I left it out of the usage in this case. It's in the example though.